### PR TITLE
Improve wording to include a wider range of people

### PR DESCRIPTION
**Pull request by lexika979**

### CODE REVIEW

**Review by 0atman — APPROVED** [review]
Lovely clarification - this was our intent from the start, thank you!

**Review by laundmo — CHANGES_REQUESTED** [review]
This leads to a paradox of tolerance: "everyone" taken literally includes transphobes, facists, etc. exactly the kind of people we don't want.

the opposite of a safe space isn't a unsafe space, it's just a space. everyone not listed in that is welcome.

but we want to be clear that these specific groups, which are often discriminated against, we try to pay special attention to so that this kind of discrimination doesn't happen.

**Comment by lexika979** [comment]
> This leads to a paradox of tolerance: "everyone" taken literally includes transphobes, facists, etc. exactly the kind of people we don't want.
So here "everyone" includes transphobes, fascists, etc

> the opposite of a safe space isn't a unsafe space, it's just a space. everyone not listed in that is welcome.
But here "everyone" is welcome?

From how I understand it you're saying that "everyone" only includes bad people when you use the word directly, which does not make sense to me, could you please elaborate?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Talking about them from a outside perspective is allowed, for example discussing
 
 ### Safe Space
 
-This community is explicitly a safe space for the following people:
+This community is explicitly a safe space for marginalised and under-represented groups of people such as:
```
Comment (0atman) [inline CODE_OF_CONDUCT.md:43]: @lexika979 Laund & I had a chat on a side channel and we figured out modified language, what do you think of this?

**Comment by laundmo** [comment]
> From how I understand it you're saying that "everyone" only includes bad people when you use the word directly, which does not make sense to me, could you please elaborate?

it's the difference between "we allow <group> to join, but follow the rules" and "it's a safe space for <group>"

a safe space specifically means that despite their often marginalised status, a group is welcome. this doesn't apply to "everyone" as "everyone" isn't a marginalised group.

**Comment by lexika979** [comment]
> > From how I understand it you're saying that "everyone" only includes bad people when you use the word directly, which does not make sense to me, could you please elaborate?
> 
> it's the difference between "we allow to join, but follow the rules" and "it's a safe space for "
> 
> a safe space specifically means that despite their often marginalised status, a group is welcome, accepted, and cared for. this doesn't apply to "everyone" as "everyone" isn't a marginalised group.

A safe space is not just for marginalized groups. It's a general term that implies acceptance for differences across the board. 
Screenshot_20230727-095036

I think it's important that we stay on track here and focus on the goal we both share, as I feel like this conversation won't help any of us much. We both want everyone to feel welcome and comfortable on the channels this code of conduct applies to.
You said that if we explicitly include "everyone", then that would include the people you don't want this to be a safe space for, which I disagree with, but understand.
I respect your opinion and want to throw in this idea:

How about we use yours and Tris's suggestion, but add a ", but not limited to, " to it? That way we don't explicitly include the people this does not apply to, but we leave room for groups not mentioned and people that aren't marginalized too. Sounds like a good way of handling this to me - What do you think?

**Comment by laundmo** [comment]
Ultima it's the responsibility of the moderators to create a safe space. it doesn't hust happen, it needs a certain level of knowledge about, and specific accomodations for, the group for which it is a safe space.

just as we can't include everyone, we also can't include groups which we don't know how to accommodate.

this is why I'm generally against anything which opens up the safe space to more than specific listed groups.

again, a safe space is meant to be specific to a group. the term safe space originated from LGBTQ+ where certain spaces were kept specifically free of harassment and discrimination against queer folx. the term inherently needs to talk about specific marginalised groups, which is why I'm also not quite happy with Tris recommendation.

is there a specific group you feel should be listed?

**Comment by lexika979** [comment]
> Ultimately it's the responsibility of the moderators to create a safe space. it doesn't just happen, it needs a certain level of knowledge about, and specific accomodations for, the group for which it is a safe space.
> 
> just as we can't include everyone, we also can't include groups which we don't know how to accommodate.
> 
> this is why I'm generally against anything which opens up the safe space clause to more than specific listed groups which we feel we can accommodate.
> 
> again, a safe space is meant to be specific to a group. the term safe space originated from LGBTQ+ where certain spaces were kept specifically free of harassment and discrimination against queer folx. the term inherently needs to talk about specific marginalised groups, which is why I'm also not quite happy with Tris recommendation.
> 
> is there a specific group you feel should be listed?

Okay, it appears you and me have very different definitions of a safe space, and how/when/where is should be used, and it feels like to me that your not ready to accept a compromise in that regard, which is fine.
How about we leave the safe space definition alone, and accommodate the idea I have into a separate section of the code if conduct? Would you be happy with that?

**Comment by laundmo** [comment]
i think that would be better yeah.

I'm not sure what your goal is beyond your understanding of safe space being different from the one i know, so maybe formulating it as a separate thing could help me understand and maybe include it.

**Comment by lexika979** [comment]
Oxford's definition of a safe space is "a place or environment in which a person or category of people can feel confident that they will not be exposed to discrimination, criticism, harassment, or any other emotional or physical harm."
This is pretty much what I'd like to include. A sentence or two which describes that no matter how "different" a person may be, they can feel welcome and will not face descrimination or harassment for it. A good example would be people that have a lisp.

**Comment by laundmo** [comment]
how do you distinguish "different" in the sense of Lisp, physical disability, etc. from "different" in the religious extremist, conspiracy theorist,
 or narcissistic way?

**Comment by lexika979** [comment]
I think regardless of one's personal opinions and believes, no one should face that kind of stuff. And as long as they oblige to the rules and don't make any other members uncomfortable, why should they not be welcome?

**Comment by lexika979** [comment]
I understand that with your definition of a safe space, you don't want to include "everyone" because it would also include the bad people, and thus tolerate their opinions. This does not tolerate their opinions, but tells them that as long as they behave by the rules and don't make anyone uncomfortable they can feel just as welcome and comfy as anyone else

**Comment by laundmo** [comment]
> Should that person fear discrimination, criticism, harassment, or any other emotional or physical harm?

no to all but criticism.

> I think regardless of one's personal opinions and believes, no one should face that kind of stuff. And as long as they oblige to the rules and don't make any other members uncomfortable, why should they not be welcome?

this is missing the point: as i said, the opposite of a safe space is not a unsafe space, but just a space. conspiracy theorists wont recieve extra attention beyond any other user to make sure that the space is safe for them. only the listed groups will.

not being listed in the safe space has absoultely no effect on how other rules are enforced. the only thing that happens because a group isnt listed in the safe space part is that we dont promise to pay special attention to making that group comfortable.

ultimately, a safe space is something we as moderators have to put work into. and thats all this section says: we'll put work into making sure these groups feel safe beyond the enforcing of normal rules. this includes things like undestanding how subtle discrimination against the group works or trying to accomodate the group to make them more comfortable (pluralkit for example)

**Comment by laundmo** [comment]
simply put: i will not agree to putting in extra effort for a unspecified amount of groups.

**Comment by lexika979** [comment]
> simply put: i will not agree to putting in extra effort for a unspecified amount of groups.

When did I ever request that you do that? I just wanted to add a section into the code of conduct that says that no matter the personal believes or differences one might have, they will not be exposed to discrimination, criticism, harassment, or any other emotional or physical harm. 

> no to all but criticism.

So you would criticise someone publicly for their opinion they never voiced to you or any other person in the space the code of conduct applies to?

**Comment by lexika979** [comment]
Snippets from the [official rust code of conduct](https://www.rust-lang.org/policies/code-of-conduct):
"We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, **or other similar characteristic**."
"Respect that people have differences of opinion [...]"

This is exactly what we need in this code of conduct.

**Comment by laundmo** [comment]
I feel like the following sections are suffecient to explain that everyone is treated well

https://github.com/0atman/noboilerplate/blob/c895449ba434077008f315596a014da492898999/CODE_OF_CONDUCT.md#L50-L60

https://github.com/0atman/noboilerplate/blob/c895449ba434077008f315596a014da492898999/CODE_OF_CONDUCT.md#L18

**Comment by laundmo** [comment]
i'm generally open to clarifying what the safe spaces section is no, its not a list of the only groups we care about. its specifically the limited list of things we choose to accomodate and pay extra attention to.

i feel like this is a core misunderstanding here, and if you have ideas how to alleviate such a misunderstanding in the future, i'd love to incorporate it.

**Comment by lexika979** [comment]
> I feel like the following sections are suffecient to explain that everyone is treated well
> 
> https://github.com/0atman/noboilerplate/blob/c895449ba434077008f315596a014da492898999/CODE_OF_CONDUCT.md#L50-L60
> 
> https://github.com/0atman/noboilerplate/blob/c895449ba434077008f315596a014da492898999/CODE_OF_CONDUCT.md#L18

Dont you see how this is flawed? This would imply that, since you "feel like the following sections are suffecient to explain that everyone is treated well", we dont need the "safe space" section.

> An important thing for my server is that it's _not_ a free-for-all, I DON'T respect the opinion of transhobes, homophobes, bigots, etc - it's not twitter by design.

I completely understand what you mean, and my goal with this is not to allow these people to make others uncomfortable or somehow add a loophole so they can go wild on the discord. My goal is to clarify that, **as long as a person follows the rules already specified in the code of conduct**, they will be treated with a base level of respect (they will not be exposed to discrimination, criticism, harassment, or any other emotional or physical harm). That is all. Nothing more to it. Really.

Somehow, even though Tris's discord server seems to be the one with the most moderation, rules, and general awareness of differences i've ever been on, it has also become the one i feel most uncomforable on.
I feel like once i say something that does not align with the opinions or believes held by some people in this space, everyone will try to invalidate my opinions and arguments by any means possible, with little sense for reason. I never ever had bad intent with any of my contributions to either conversations on the discord, or this code of conduct. I've been putting a exceptional amount of time into these things and have felt almost nothing but disregard and contempt in return.
Hell, i've even made sure to double-check everything in this PR **before** i even made it. I googled the official definitions (like "including, but not limited to" or "safe space") because english is not my main language, and designed my suggestions around those definitions.
I have tried to tried my absolute best to contribute in the most clear and simple way possible. I've tried to be the most accepting possible within reason, but somehow failed again.

I will close this and leave the discord server. I am genuinely sorry for the inconvenience i've caused, i just cant seem to understand this.
@laundmo I'd recommend to document the goals and definitions for things used in this code of conduct if they differ from the official ones. That would make the process of contributing for other people alot simpler and more painless.

Thank you two for the conversation, i respect both of you and apologize if that didnt come across correctly.
But for my own mental health, i will not put any more of my time into this right now.

**Comment by 0atman** [comment]
@lexika979 I'm sorry this has become stressful, thank you for your help

**Pull request closed without merging**